package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^5.53.0",
+    "eslint": "^8.34.0",
+    "eslint-config-standard-with-typescript": "^34.0.0",
+    "eslint-plugin-import": "^2.27.5",
+    "eslint-plugin-n": "^15.6.1",
+    "eslint-plugin-promise": "^6.1.1",
+    "eslint-plugin-react": "^7.32.2",
```

Comment:
I'm not 100% sure if these are being used? I would imagine they have to be imported / configured in your `.eslintrc.js` but it's possible that you're using  something that already includes these and these are just peer dependencies.